CONTRIBUTING.rst

```diff
@@ -190,6 +190,11 @@ Running System Tests
    $ export CLOUDSDK_PYTHON_SITEPACKAGES=1
 
    # Authenticate the gcloud tool with your account.
+   $ P12_CREDENTIALS_FILE="path/to/app_credentials.json"
```

Comment:
Yeah, I also noticed after I submitted this that P12 is deprecated according to here https://cloud.google.com/sdk/crypto.  I can look into changing the variable name.